File: CPU_2025/Gato/gato.py
import time

inicio = time.perf_counter()

casa = [int(item) for item in input().split(" ")]
gato = [int(item) for item in input().split(" ")]

if((gato[0] - casa[0])**2 + (gato[1] - casa[1])**2 <= casa[2]**2):
	print("Gato")
else:
	print("Gaiato")

fim = time.perf_counter()

print(fim - inicio)

File: CPU_2025/Gato/gato.cpp
#include <iostream>
using namespace std;

int main() {

	int x1, x2, y1, y2, r;
	cin >> x1 >> y1 >> r;
	cin >> x2 >> y2;

	if((x1 - x2) * (x1 - x2) + (y1 - y2) * (y1 - y2) <= r * r)
		cout << "Gato" << endl;
	else
		cout << "Gaiato" << endl;

	return 0;
}
